loader/loader.go

```diff
@@ -536,13 +571,24 @@ func (l *Loader) Restore(ctx context.Context) error {
 
 	go l.PrintStatus(ctx)
 
-	if err := l.restoreData(ctx); err != nil {
-		if errors.Cause(err) == context.Canceled {
-			return nil
-		}
+	begin := time.Now()
+	err = l.restoreData(ctx)
+	if err != nil && errors.Cause(err) != context.Canceled {
```

Comment:
Is it needed
```
l.closeFileJobQueue()
l.workerWg.Wait()
```
before `return err`?

Reply:
I thought it, but I don't change the origin logic, let it be